client/src/components/FilterForm.tsx

```diff
@@ -49,7 +49,7 @@ export default function FilterForm({
             <option value={''}>Location</option>
             <option value={'San Francisco, CA'}>San Francisco, CA</option>
             <option value={'Bellevue, WA'}>Bellevue, WA</option>
-            <option value={'Austin, TX'}>Austin, TX</option>
+            <option value={'New York, NY'}>New York, NY</option>
```

Comment:
FYI, if you end up actually using this in your job search, you may find it nicer to build this list dynamically. You could either add a table to your database that has this list in it, or you could query the jobs table for all the unique locations and use that to build this list.

In fact, given that this is a filter in your frontend, you might be able to build the list from the data you already have in your frontend. That's usually how these filter drop-downs are built.